blocklylib-core/src/main/assets/background_compiler.html

```diff
@@ -40,14 +48,15 @@
         }
 
         import_generators(BlocklyJavascriptInterface.getBlockGeneratorsFilenames());
+        import_generator_language(BlocklyJavascriptInterface.getGeneratorLanguageFilename());
```

Comment:
Why not import the default language specific generating script before the user defined ones?
Maybe someone wants to override the functions defined in the default generator.